extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -657,7 +657,9 @@ export class ProjectsController {
 			let fileFolderList: string[] = model.extractTarget === mssql.ExtractTarget.file ? [model.filePath] : await this.generateList(model.filePath); // Create a list of all the files and directories to be added to project
 
 			await project.addToProject(fileFolderList); // Add generated file structure to the project
-			await this.openProject(vscode.Uri.file(newProjFilePath));
+
```

Comment:
I think we should rename this method and the command to : createProjectFromDatabase

Reply:
renamed all the remaining "import database" to create project from database